#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_lintian_override.dpatch by  <edd@xmcorsairs.wu-wien.ac.at>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add a lintian override file

@DPATCH@

diff -ruN RJaCGH.orig/debian/RJaCGH.lintian-overrides RJaCGH/debian/RJaCGH.lintian-overrides
--- RJaCGH.orig/debian/RJaCGH.lintian-overrides	1970-01-01 01:00:00.000000000 +0100
+++ RJaCGH/debian/RJaCGH.lintian-overrides	2009-05-17 21:09:51.000000000 +0200
@@ -0,0 +1 @@
+r-cran-rjacgh: embedded-zlib ./usr/lib/R/site-library/RJaCGH/libs/RJaCGH.so
